d2renderers/d2svg/class.go

```diff
@@ -3,17 +3,21 @@ package d2svg
 import (
 	"fmt"
 	"io"
-	"strings"
 
 	"oss.terrastruct.com/d2/d2target"
 	"oss.terrastruct.com/d2/lib/geo"
 	"oss.terrastruct.com/d2/lib/label"
 	"oss.terrastruct.com/d2/lib/svg"
+	svg_style "oss.terrastruct.com/d2/lib/svg/style"
```

Comment:
go conventions:
1. no underscore imports (just combine the words all lowercase)
2. for subpackages, if ambiguous, just repeat suffix. e.g. `svgstyle`

Reply:
done